files/ceph-version-qemu.sh

```diff
@@ -3,38 +3,82 @@
 # By default, search for upgrades from Jewel to Luminous.
 CEPH_MAJORVER_EXPECTED_OLD=10
 CEPH_MAJORVER_EXPECTED_NEW=12
+# By default, exclude VMs that do not have RBD block devices attached.
+EXCLUDE_QEMU_WITHOUT_RBD=0
+DEBUG=1
 
-# Expect that Ceph was upgraded from major version $1
-if [ -z ${CEPH_MAJORVER_EXPECTED_OLD+x} ]; then
-  CEPH_MAJORVER_EXPECTED_OLD=$1
-fi
-# Expect that Ceph was upgraded from major version $2
-if [ -z ${CEPH_MAJORVER_EXPECTED_NEW+x} ]; then
-  CEPH_MAJORVER_EXPECTED_NEW=$2
-fi
+usage() { echo "Usage: $0 [-o 10] [-n 12] [-e] [-d]" 1>&2; exit 1; }
```

Comment:
Probably better documentation would be welcomed.

Reply:
Added